### DISCUSSION TITLE
Close All windows

### DISCUSSION THREAD

**Discussion by MehanikTMYT** [AvaloniaUI/Avalonia#14803] [Q&A]
How to close all windows without sending Close() to all windows. For example, I have 4 windows (MainWindow SettingsWindow LogsWindow VersionWindow), how to close all of them by closing one of them

**Comment by jp2masa** [comment]
You can use `OnMainWindowClose` shutdown mode: [docs](https://docs.avaloniaui.net/docs/concepts/application-lifetimes#iclassicdesktopstyleapplicationlifetime).
